<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Doctor's end software</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>WELCOME</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NFCButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>270</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;
color:black;
border-style:outset;
border-width:2px;
border-color:red;
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Click to scan NFC</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="Next_doc">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>400</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:blue;
color:white;
border-style:outset;
border-width:2px;
border-color:white;
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>E:/Capstone/sym_gui/forward.jpg</normaloff>E:/Capstone/sym_gui/forward.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="Exit_doc">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>480</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:blue;
color:white;
border-style:outset;
border-width:2px;
border-color:white;
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>E:/Capstone/sym_gui/forward.jpg</normaloff>E:/Capstone/sym_gui/forward.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="DateTime1">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
